examples/graphql-returning-record-values/graphql_returning_record_values.md

```diff
@@ -1,10 +1,10 @@
 # GraphQL service - Record as output object
 
-In a Ballerina GraphQL service, a record type can be used to represent an `OBJECT` type in a GraphQL schema. The type name and the field names are mapped one-to-one from the Ballerina to GraphQL. Types are generated by recursively analyzing the types that are used in a Ballerina GraphQL service.
+In a Ballerina GraphQL service, a record type can be used to represent an `OBJECT` type in a GraphQL schema. The type name and the field names are mapped one-to-one from the Ballerina to GraphQL. Types are generated by analyzing the types that are used in a Ballerina GraphQL service recursively. Use a record type to represent an `OBJECT` type only when all fields of that object type do not have any input arguments or the field resolution does not require any complex logic execution.
 
```

Comment:
In a Ballerina GraphQL service, a record type can be used to represent an `OBJECT` type in a GraphQL schema. The type name and the field names are mapped one-to-one from the Ballerina to GraphQL.  These `records` can be returned from the `resource` or `remote` methods as GraphQL output `OBJECT`. Use a record type to represent an `OBJECT` type only when all fields of that object type do not have any input arguments or the field resolution does not require any complex logic execution. `record` type is preferred over `object` type, in this case, as it makes code more concise.